google/cloud/datacatalog/v1beta1/datacatalog_gapic.yaml

```diff
@@ -156,6 +156,50 @@ interfaces:
     retry_codes_name: idempotent
     retry_params_name: default
     timeout_millis: 60000
+    samples:
+      standalone:
+      - region_tag: data_catalog_lookup_entry
+        value_sets: [data_catalog_lookup_entry]
+      - region_tag: data_catalog_lookup_entry_sql_resource
+        value_sets: [data_catalog_lookup_entry_sql_resource]
+    sample_value_sets:
+    - id: data_catalog_lookup_entry
+      description: Lookup Entry
+      parameters:
+        defaults:
+        - linked_resource = "[Full Resource Name]"
```

Comment:
Could you use an example resource name, similar to one of the ones you used in the Python sample? It's helpful for developers to see what a resource name looks like.

Reply:
Done.